components/widgets/menu/menu_items/menu_item.scss

```diff
@@ -354,3 +354,30 @@
     border-top: $border-gray;
     margin: 8px 0;
 }
+
+.destructive-leave-channel {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 5px 0;
+    color: var(--dnd-indicator);
+    gap: 9px;
+
+    &:hover {
+        background-color: var(--dnd-indicator);
+        color: #fff;
+    }
+
+    @media (max-width: 768px) {
+        justify-content: start;
+        padding-left: 14px;
+    }
+}
+
+.destructive-leave-channel:hover .text-white {
+    color: #fff !important;
+}
+
+.text-white {
+    color: var(--dnd-indicator) !important;
+}
```

Comment:
when using the `Menu.ItemAction` with the `isDangerous` prop all of those styles are not needed

Reply:
screenshot of the `Delete Category` item in the category dropdown menu in the LHS

![Screenshot 2022-08-30 at 14 23 17](https://user-images.githubusercontent.com/32863416/187435647-e2a8b566-b74b-4f4c-8b84-3a0c04820a6b.png)